components/ad/smartbanner/src/index.js

```diff
@@ -44,16 +44,21 @@ class AdSmartbanner extends Component {
 AdSmartbanner.propTypes = {
   onClick: PropTypes.func.isRequired,
   onClose: PropTypes.func.isRequired,
-  icon: PropTypes.func.isRequired,
+  icon: PropTypes.func,
   imageUrl: PropTypes.string.isRequired,
   title: PropTypes.string.isRequired,
   text: PropTypes.string.isRequired,
   buttonText: PropTypes.string.isRequired,
-  staticPosition: PropTypes.bool
+  staticPosition: PropTypes.bool,
+  customRatingIcons: PropTypes.obj,
```

Comment:
What about specifying the object shape here?

Reply:
Ok, added.

Reply:
Pero el shape ya está definido en el componente que requiere la prop, aquí en realidad no es necesario.